homeassistant/components/media_player/mediaroom.py

```diff
@@ -46,18 +46,21 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Mediaroom platform."""
     hosts = []
 
+    if hass.data.get(DATA_MEDIAROOM) is None:
```

Comment:
```py
known_hosts = hass.data.get(DATA_MEDIAROOM)
if known_hosts is None:
   known_hosts = hass.data[DATA_MEDIAROOM] = []
```
Then use known_hosts below so you don't need to access the dict multiple times.

Reply:
I was just following the pattern on the Yamaha media_player :)

Care to enlighten me on why doing this change ? (I'm submitting a new commit with the change)

Reply:
Caching the list in a variable avoids having to access the dict multiple times, so it's a small performance clean up.

Reply:
dict is key indexed, don't think it makes any difference. But the change has been submitted.